services/venice-router/src/main/java/com/linkedin/venice/router/RouterServer.java

```diff
@@ -518,13 +525,19 @@ public boolean startInner() throws Exception {
         config.getClusterName(),
         compressorFactory,
         metricsRepository);
+
+    retryManagerExecutorService =
```

Comment:
Quick question:
how do we pick up the thread number: `3` here?
Does it need to be tunable?

Reply:
3 is arbitrary, will make a config for this.